src/test/java/lotto/CheckerTests.java

```diff
@@ -0,0 +1,34 @@
+package lotto;
+
+import lotto.model.Checker;
+import lotto.model.GameCount;
+import lotto.model.Games;
+import lotto.model.WinningNumbers;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.LinkedHashMap;
+
+import static lotto.common.Constants.GET_NUMBER_COUNT;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("결과확인 기능 테스트")
+public class CheckerTests {
+
+    @DisplayName("결과확인을 테스트합니다.")
+    @Test
+    public void 결과확인() {
+        int purchaseAmount = 3000;
+        GameCount gameCount = new GameCount(purchaseAmount);
+        Games games = new Games(gameCount.getValue());
+        WinningNumbers winningNumbers = new WinningNumbers("1,2,3,4,5,6");
+        Checker checker = new Checker(games, winningNumbers);
+
+        assertThat(checker.getResults())
+                .isInstanceOf(LinkedHashMap.class)
+                .isNotEmpty();
+        assertThat(checker.getResults().size()).isEqualTo(GET_NUMBER_COUNT + 1);
```

Comment:
```suggestion
assertAll(
    () -> assertThat(checker.getResults())
                .isInstanceOf(LinkedHashMap.class)
                .isNotEmpty(),
    () -> assertThat(checker.getResults().size()).isEqualTo(GET_NUMBER_COUNT + 1)
);
```
하나의 테스트에서 두개 이상의 검증을 해야 할 경우 `assertAll` 로 묶어줄 수 있습니다!
관련해서는 assertAll 에 대한 내용을 찾아보시면 좋을 것 같아요 ~
하나씩 검증하는것과 동작에 대한 차이가 조금 있습니다 !

참고: https://sas-study.tistory.com/316, https://escapefromcoding.tistory.com/358?category=1184809

Reply:
제가 간과한 부분이었습니다.😅
말씀하신대로 assertAll을 사용하도록 하겠습니다.